graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -278,9 +371,8 @@ def _ensure_no_cascading_type_suppressions(
                     )
                 )
             error_message_components.append(
-                "A schema containing a field that is of a nonexistent type is invalid. When field "
-                "suppression is supported, you can fix this problem by suppressing the fields "
-                "shown above."
+                "A schema containing a field that is of a nonexistent type is invalid. To fix "
+                "this, suppress the fields using the field_renamings argument of rename_schema."
```

Comment:
Yep! The description is just above but not shown in the diff because it wasn't changed-- I'll change this error message slightly to clarify which fields should be changed.